### PR TITLE
Updating getting-started

### LINKED ISSUE #45: Rename repo and update docs
As a developer, I'd like to rename the repo from `spring-cloud-dataflow-admin-mesos` to `spring-cloud-dataflow-server-mesos`.

### PR DESCRIPTION
**Pull request by trisberg**
Resolves #45

### CODE REVIEW

File: spring-cloud-dataflow-server-mesos-docs/src/main/asciidoc/getting-started.adoc [hunk 1]
```diff
@@ -30,32 +30,38 @@ $ redis-sever
 +
 This is used by the locally running Data Flow Server to store the state of available module versions for stream definitions.
 +
-NOTE: If you are switching between milestone and snapshot versions of the Data Flow Server, flush the redis keys that contain the module's version information.  Calling `flushdb` is indiscriminate but a handy way to start from a clean redis state.
+. Download the Spring Cloud Data Flow Server for Mesos and Marathon.
 +
-. Download and run the Spring Cloud Data Flow Server for Mesos and Marathon.
+```
+$ wget http://repo.spring.io/snapshot/org/springframework/cloud/spring-cloud-dataflow-server-mesos/1.0.0.M2/spring-cloud-dataflow-server-mesos-1.0.0.M2.jar
```
Comment (cppwfs) [inline spring-cloud-dataflow-server-mesos-docs/src/main/asciidoc/getting-started.adoc:36]: should that be milestone instead of snapshot

**Comment by cppwfs** [comment]
`Spring Cloud Data Flow Admin for Mesos` in index.adoc probably needs to be `Spring Cloud Data Flow Server for Mesos`

**Comment by cppwfs** [comment]
Modules section is not being rendered correctly :-(

Is the URL for scsm-asciidoc still correct, or should we pointing to starters?

File: spring-cloud-dataflow-server-mesos-docs/src/main/asciidoc/getting-started.adoc (hunk 1 above)
Comment (trisberg) [inline spring-cloud-dataflow-server-mesos-docs/src/main/asciidoc/getting-started.adoc:36]: yes, it should be milestone

**Comment by cppwfs** [comment]
To fix the modules section we need to replace scsm-asciidoc and its url in the index.adoc with `:scstas-asciidoc: https://raw.githubusercontent.com/spring-cloud/spring-cloud-stream-app-starters/master/spring-cloud-stream-app-starters-docs/src/main/asciidoc` .
Then update the scsm-asciidoc with sctas-asciidoc in module.adoc

**Comment by cppwfs** [comment]
Fixed issues above, squashed merged.

**Pull request closed without merging**